### ISSUE TITLE
Display some additional info for each Collective

### ISSUE THREAD

**Issue by PatrikD** [sudoroom/sudo-humans#70]
Allow each Collective to display some additional information on their front page, e.g. mission statement, logo, links to where folks can find out more about the membership process.

Right now, we have people signing up for membership through sudo humans, but then they don't know they also need to sign our membership agreement form, go through a 1 month waiting period, take a safety quiz, etc.

**Comment by rcsheets** [comment]
Are any of those extra steps things we could potentially integrate into humans? For example, we could display an agreement (but can't make anyone actually read it).

**Comment by PatrikD** [comment]
Definitely. We have a membership agreement form that we're currently asking people to print out and sign. We should be able to easily turn that into a fillable Google form that we could link people to, and then track  on humans whether they've filled it out.

Likewise, we have an online safety training quiz that's already in a google form. Not required from all members, but it would be great if we could display their highest score along with all other member info.

We also have an application form for sponsored membership (another google form that dumps results into a google spreadsheet). Once we approve their application, that essentially gives them 6 months of reduced membership fee.

Lastly, our membership process includes a 1 month trial period after signing the membership agreement form and setting up a payment plan on humans (or getting approval for sponsored membership). After that one month trial period they are supposed to get two existing members to vouch for them - which is a detail that often gets skipped completely because by that time we're already used to seeing them around, nobody checks the 1 month deadline anyway, and we don't have any processes to collect those member vouchings anyway. Sigh... our process is a royal mess!
